core/wiki/tags/ControlPanelPlugins.tid

```diff
@@ -1,2 +1,2 @@
 title: $:/tags/ControlPanel/Plugins
-list: [[$:/core/ui/ControlPanel/Plugins/Installed]] [[$:/core/ui/ControlPanel/Plugins/Add]]
+list: $:/core/ui/ControlPanel/Plugins/Installed/Plugins $:/core/ui/ControlPanel/Plugins/Installed/Themes $:/core/ui/ControlPanel/Plugins/Installed/Languages [[$:/core/ui/ControlPanel/Plugins/Installed]] [[$:/core/ui/ControlPanel/Plugins/Add]]
```

Comment:
I could not find where the `$:/tags/ControlPanel/Plugins` is used, so I did not delete the last two items in the list field `[[$:/core/ui/ControlPanel/Plugins/Installed]] [[$:/core/ui/ControlPanel/Plugins/Add]]`

Reply:
Those entries are left over from an earlier commit 8643278a452d1a300cec8d3425c1b18699a17dca and can be removed

Reply:
Thanks @Jermolene 
Should I prepare any documentation and submit to tw5.com?
I do not see any docs out there to edit/revise!

Reply:
Hi @kookma we should add a reference to the new tag in the "SystemTags" documentation tiddler.

Reply:
You should be able to add the docs update to this PR (docs updates that accompany core changes go to the master branch, not tiddlywiki-com).

Reply:
Sure, I check and update the SystemTags documentation tiddler